test/integration/vshard_api_test.lua

```diff
@@ -0,0 +1,73 @@
+local fio = require('fio')
+local t = require('luatest')
+local g = t.group()
+
+local helpers = require('test.helper')
+
+g.before_all = function()
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        server_command = helpers.entrypoint('srv_basic'),
+        cookie = require('digest').urandom(6):hex(),
+        replicasets = {
+            {
+                uuid = helpers.uuid('a'),
+                roles = {'vshard-router'},
+                servers = {{
+                    alias = 'A1',
+                    instance_uuid = helpers.uuid('a', 'a', 1),
+                    advertise_port = 13301,
+                    http_port = 8081,
+                }}
+            }, {
+                uuid = helpers.uuid('b'),
+                roles = {'myrole'},
+                servers = {
+                    {
+                        alias = 'B1',
+                        instance_uuid = helpers.uuid('b', 'b', 1),
+                        advertise_port = 13302,
+                        http_port = 8082,
+                    },{
+                        alias = 'B2',
+                        instance_uuid = helpers.uuid('b', 'b', 2),
+                        advertise_port = 13303,
+                        http_port = 8083,
+                    }
+                }
+            }
+        }
+    })
+    g.cluster:start()
+end
+
+g.after_all = function()
+    g.cluster:stop()
+    fio.rmtree(g.cluster.datadir)
+end
+
+function g.test_get_cfg()
```

Comment:
please add a multisharding test. (see example in tests with name multisharding*)